bbn-test-4/finality-providers/README.md

```diff
@@ -226,10 +231,38 @@ being the same as the finality provider information stored under `./finality-pro
 but with the `.sig` extension (e.g. `${nickname}.sig`).
 The content of the file should be the plain value of the `schnorr_signature_hex` field.
 
+The signature was generated by reading the entire file data, not only the file
+content. For proper verification, the exactly file used for signing should
```

Comment:
```suggestion
content. For proper verification, the exact file used for signing should
```

Let's also highlight this with a warning etc?

Reply:
https://github.com/babylonchain/networks/pull/193/commits/8b52ad3dcd8691a2934708490b1f24e95460d8bd

Reply:
https://github.com/babylonchain/networks/pull/193/commits/e680ef1cba6f593c4a4907264b45a07c0902e95a